docs/content/preview/quick-start/build-apps/nodejs/ysql-sequelize.md

```diff
@@ -61,6 +62,24 @@ $ cd ./node/sequelize/
 ```sh
 npm install
 ```
+## Specifying SSL configuration:
+
+This configuration can be used while connecting to a YB Managed cluster or a local YB cluster with SSL enabled.
+
+- Use the configuration in the following way in `models/index.js` file while creating sequelize object:
+```
+sequelize = new Sequelize("<db_name>", "<user_name>","<password>" , {
+    dialect: 'postgres',
+    port: 5433,
+    host: "<host_name>",
+    dialectOptions: {
+        ssl: {
+            rejectUnauthorised: true,
+            ca: fs.readFileSync('<path_to_root_crt>').toString(),
+        }
+    }
+  });
+```
```

Comment:
Markdownlint is helpful, and we have rules already defined for it in the repo...

(And again, should it be unauthoriZed?)

````suggestion

## Specifying SSL configuration

This configuration can be used while connecting to a YB Managed cluster or a local YB cluster with SSL enabled.

Use the configuration in the following way in the `models/index.js` file when you create the sequelize object:

```js
sequelize = new Sequelize("<db_name>", "<user_name>","<password>" , {
    dialect: 'postgres',
    port: 5433,
    host: "<host_name>",
    dialectOptions: {
        ssl: {
            rejectUnauthorised: true,
            ca: fs.readFileSync('<path_to_root_crt>').toString(),
        }
    }
  });
```
````